XCGLogger/Library/XCGLogger/XCGLogWrapper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  XCGLogWrapper.swift
+//  XCGLogger
+//
+//  Created by Anders Borch on 3/7/16.
+//  Copyright © 2016 Cerebral Gardens. All rights reserved.
+//
+
+import Foundation
+
+@objc public enum LogLevel: Int {
```

Comment:
This could probably be namespaced to avoid collisions in the objc namespace.
Also to avoid confusing when using from Swift, since there's already a type for this, is there a way to hide this from Swift?